scss/jeet/_functions.scss

```diff
@@ -1,23 +1,20 @@
-/**
- * List functions courtesy of the wonderful folks at Team Sass.
- * Check out their awesome grid: Singularity.
- */
-
-/**
- * Get  percentage from a given ratio.
- * @param {number} [$ratio=1] - The column ratio of the element.
- * @returns {number} - The percentage value.
- */
+/// List functions courtesy of the wonderful folks at Team Sass.
```

Comment:
What should we use instead? Just double slash? I just triple slashed every single comment in a big project I'm working on..........